dlk/hw/make_os/build/setting_after_chroot.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+/debootstrap/debootstrap --second-stage # After chroot
+echo "ubuntu" > /etc/hostname # After chroot
+useradd ubuntu --create-home --shell /bin/bash # After chroot
+echo 'ubuntu:ubuntu' | chpasswd # After chroot
```

Comment:
> `# After chroot`

This comment is meanless I think. For example, it is better to set a comment above to explain that `this script should be run with chroot`.